opendevin/core/config.py

```diff
@@ -63,6 +64,9 @@ class LLMConfig(metaclass=Singleton):
     retry_max_wait: int = 60
     timeout: int | None = None
     max_chars: int = 5_000_000  # fallback for token counting
+    max_message_chars: int = (
+        10000  # maximum number of characters in a message to the llm
+    )
```

Comment:
This seems pretty small... May source files seem like they would be larger than this. I wonder how often this is triggered, and if it has any effect on accuracy.

Reply:
It's the default we're currently using on `main` https://github.com/OpenDevin/OpenDevin/blob/009f6f9ebc24b8cbce239f0929ac9c25bd3b055c/agenthub/codeact_agent/codeact_agent.py#L90

FWIW I made it 100,000 on my machine, once it was configurable here. Partly because of some work with PDFs, for those 10k is very low, and it was triggered all the time.

Should we try a higher default, like, maybe not 100k, but some 25k ?